source/index.html.md

```diff
@@ -1193,6 +1195,9 @@ POST https://gateway.clearhaus.com/credits
 <p class="alert alert-info">
 <b>Notice:</b> Implicitly, <code>initiator</code> is <code>merchant</code> and
 <code>credential_on_file</code> is <code>use</code>.
+<br />
+<b>Notice:</b> SCA is required when <code>credential_on_file</code> is
+<code>store</code> when the card is in scope for SCA.
```

Comment:
```suggestion
<b>Notice:</b> SCA is required when <code>credential_on_file</code> is
<code>store</code> and the card is in scope for SCA.
```

To avoid repeating _when_.